Bento/Views/BentoReusableView.swift

```diff
@@ -1,10 +1,20 @@
+/// Existential type for reusable views (cell, header, footer) in `UITableView`, `UICollectionView`, etc.
 protocol BentoReusableView: AnyObject {
-    var containedView: UIView? { get set }
+    /// The main view to which you add your reusable view’s custom content.
```

Comment:
```suggestion
    /// The view to which `containedView` should be added.
```

Reply:
Proposed edition provides less information about `contentView`.
If we see `containedView`'s comment, it already provides the hierarchy, so we don't need to mention twice.

(BTW, this comment is taken from UIKit's original comment, which I think is reasonable)

Reply:
> Proposed edition provides less information about contentView.

In the context of `BentoReusableView`, the `contentView` requirement for Bento is as simple as described — "custom content" is `containedView` and would stay as such based on today's knowledge.